frontend/src/reports/ui/DisclaimerAlert.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+import AlertTitle from "@material-ui/lab/AlertTitle";
+import Button from "@material-ui/core/Button";
+import styles from "./DisclaimerAlert.module.scss";
+import FlagIcon from "@material-ui/icons/Flag";
+
+function DisclaimerAlert(props: { jumpToData: () => void }) {
+  return (
+    <Alert
+      severity="warning"
+      className={styles.ReportAlert}
+      icon={<FlagIcon />}
+    >
+      <AlertTitle>Major gaps in the data</AlertTitle>
+      <p>
+        <span style={{ lineHeight: "21px" }}>
+          Structural racism and oppression create health inequities, which leads
+          to missing data. The maps and tables below reflect the best data we
+          have, but there are major known gaps in the data. We're working to
```

Comment:
Do we care that we are using the word "major" twice (in the title and in the text here)?

Reply:
Chi and Larry both okay'ed this copy so I'll go ahead and submit. Easy to change later! I think it wouldn't hurt to have a "copy bash" before launch for all these tweaks